README.md

```diff
@@ -14,15 +14,61 @@ It is also possible to use this library to convert an SVG to a PNG before downlo
 ```javascript
 import downloadSvg from 'svg-crowbar';
 
-downloadSvg(document.querySelector('svg'), 'my_svg');
+downloadSvg(document.querySelector('svg'));
 ```    
 or
 ```javascript
 import { downloadPng } from 'svg-crowbar';
 
-downloadPng(document.querySelector('svg'));
+downloadPng(document.querySelector('svg'), 'my_svg', { css: 'internal' });
 ```
 
-Filename is determined by element id, class or page title, when not provided explicitly.
+The `downloadSVG`/`downloadPNG` functions each have three arguments:
 
-An error is thrown in case no valid SVG element was provided.
+```javascript
+downloadSVG([svgElement], [filename], [options])
+downloadPNG([svgElement], [filename], [options])
+```
+
+- **svgElement** *(required)*
+  
+  A DOM element selector for an SVG, e.g. `document.querySelector('svg')`. An error is thrown if no valid SVG element was provided.
+
+- **filename** *(optional)*
+
+  A string to set the filename. This is determined by element id, class or page title, when not provided explicitly.
+
+- **options** *(optional)*
+
+  An object literal. It presently has just a single configurable property:
+
+- **options.css** *(optional)*
+
+  This setting determines how the SVG will be styled:
+
+  - **`'inline'`**
+
+    Default value. Inlines all computed styles on every element in the SVG. This setting best ensures that the exported SVG is accurate cross-browser.
+
+  - **`'internal'`**
+
+    Adds an internal block of styles containing only explicitly declared style rules (from `document.styleSheets`). This can drastically reduce file-sizes and build time in exported SVGs, but could be less accurate if the styles are not consistent cross-browser, as it does not include styles from the browser's user agent stylesheet.
+
+  - **`false`**
+
+    Doesn't add any CSS. This gives the smallest file-size, but you might need to manually add your own styles to exported SVGs to ensure an accurate output. You can do this by injecting a `<style>` block into the selected SVG before exporting.
+
+  Example:
+  ```javascript
+  const svg = document.querySelector('svg');
+
+  // Add inline styles on SVG elements:
+  downloadSvg(svg, 'my_svg'); 
+  downloadSvg(svg, 'my_svg', { css: 'inline' });
+
+  // Add a <style> block in the SVG:
+  downloadSvg(svg, 'my_svg', { css: 'internal' });
+
+  // Do not add CSS:
+  downloadSvg(svg, 'my_svg', { css: false });
```

Comment:
After giving it a second thought, it feels like using string value, e.g., `'none'` might be better in terms of consistency?

Reply:
Sure, I don't mind too much. tbh at the moment it doesn't matter what you provide: Any value other than `'internal'`, `'inline'`, or `undefined` will achieve the same result. But I'll change it in the docs! :)